Foundation/NSRunLoop.swift

```diff
@@ -7,38 +7,51 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+import CoreFoundation
 
-public let NSDefaultRunLoopMode: String = "NSDefaultRunLoopMode"
-public let NSRunLoopCommonModes: String = "NSRunLoopCommonModes"
+public let NSDefaultRunLoopMode: String = kCFRunLoopDefaultMode._swiftObject
+public let NSRunLoopCommonModes: String = kCFRunLoopCommonModes._swiftObject
 
 public class NSRunLoop : NSObject {
+    typealias CFType = CFRunLoopRef
+    internal var _cfObject : CFType!
```

Comment:
Let's call this var `_cfRunLoop` or similar. `_cfObject` is reserved for our toll-free bridging.